flexget/plugins/list/subtitle_list.py

```diff
@@ -0,0 +1,261 @@
+
+    def __repr__(self):
+        return '<SubtitleListList name=%s,id=%d>' % (self.name, self.id)
+
+    def to_dict(self):
+        return {
+            'id': self.id,
+            'name': self.name,
+            'added_on': self.added
+        }
+
+
+class SubtitleListFile(Base):
+    __tablename__ = 'subtitle_list_files'
+    id = Column(Integer, primary_key=True)
+    added = Column(DateTime, default=datetime.now)
+    title = Column(Unicode)
+    location = Column(Unicode)
+    list_id = Column(Integer, ForeignKey(SubtitleListList.id), nullable=False)
+    languages = relationship('SubtitleListLanguage', secondary=association_table, backref="file", lazy='joined')
+    remove_after = Column(Unicode)
+
+    def __repr__(self):
+        return '<SubtitleListFile title=%s,path=%s,list_name=%s>' % (self.title, self.location, self.list.name)
+
+    def to_entry(self):
+        entry = Entry()
+        entry['title'] = self.title
+        entry['url'] = 'mock://localhost/subtitle_list/%d' % self.id
+        entry['location'] = self.location
+        entry['subtitle_languages'] = []
+        entry['remove_after'] = self.remove_after
+        entry['added'] = self.added
+        for subtitle_language in self.languages:
+            entry['subtitle_languages'].append(subtitle_language.language)
+        return entry
+
+    def to_dict(self):
+        subtitle_languages = [subtitle_list_language.to_dict() for subtitle_list_language in self.languages]
+        return {
+            'id': self.id,
+            'added_on': self.added,
+            'title': self.title,
+            'location': self.location,
+            'subtitle_languages': subtitle_languages
+        }
+
+
+class SubtitleListLanguage(Base):
+    __tablename__ = 'subtitle_list_languages'
+    id = Column(Integer, primary_key=True)
+    added = Column(DateTime, default=datetime.now)
+    language = Column(Unicode)
+
+    def __eq__(self, other):
+        if isinstance(other, SubtitleListLanguage):
+            return self.language == other.language
+        else:
+            return False
+
+    def __ne__(self, other):
+        return (not self.__eq__(other))
+
+    def __hash__(self):
+        return hash(self.__repr__())
+
+    def __repr__(self):
+        return '<SubtitleListLanguage id=%s,language=%s>' % (self.id, self.language)
+
+    def to_dict(self):
+        return {
+            'id': self.id,
+            'added_on': self.added,
+            'language': self.language
+        }
+
+
+class SubtitleList(MutableSet):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'list': {'type': 'string'},
+            'languages': {'type': 'array', 'items': {'type': 'string'}, 'minItems': 1},
+            'check_subtitles': {'type': 'boolean', 'default': True},
+            'remove_after': {'type': 'string', 'format': 'interval'}
+        },
+        'required': ['list'],
```

Comment:
if `list` is the only required property, wouldn't it be more friendly to have the schema use `oneOf` string or object, so you could call the list with `subtitle_list: list_name`.
just a suggestion